### PR TITLE
improve ux of various builder elements

### PR DESCRIPTION
**Pull request by martenson**

### CODE REVIEW

File: client/galaxy/scripts/components/RuleCollectionBuilder.vue [hunk 1]
```diff
@@ -300,11 +300,14 @@
         </div>
         <div class="rule-footer footer flex-row no-flex vertically-spaced" v-if="ruleView == 'source'">
             <option-buttons-div>
+                <button @click="attemptRulePreview" class="btn btn-secondary rule-btn-cancel">
```
Comment (martenson) [inline client/galaxy/scripts/components/RuleCollectionBuilder.vue]: this should probably call a different function but I did not find a proper one

Comment (jmchilton) [reply client/galaxy/scripts/components/RuleCollectionBuilder.vue]: This button does the same thing as the Apply button right?

Comment (martenson) [reply client/galaxy/scripts/components/RuleCollectionBuilder.vue]: Yeah, I would like it to do proper cancel instead, hence my comment.

Comment (jmchilton) [reply client/galaxy/scripts/components/RuleCollectionBuilder.vue]: I see - I'll see if I can make that work. Probably just reset following by apply.

**Pull request closed without merging**